src/diffusers/__init__.py

```diff
@@ -128,6 +129,8 @@
             "UNetSpatioTemporalConditionModel",
             "UVit2DModel",
             "VQModel",
+            "DCAE",
+            "DCAE_HF",
```

Comment:
are these the final class names? i would have thought they'd be a bit longer and more descriptive like DeepCompressionAutoEncoder

Reply:
I will be taking over on this PR to make the relevant changes for full integration soon, so will address this then :) LMK if there's anything particular that you'd like to see